components/angular-tomitribe-select/src/tomitribe-select.ts

```diff
@@ -131,13 +131,27 @@ module tomitribe_select {
                 }
             }
 
-            // multiple selects have no focusser
-            if (uiSelect.multiple) {
-                angular.element(uiSelect.focusInput).on('focus', focusHandler);
-            } else {
-                angular.element(uiSelect.focusser).on('focus', focusHandler);
+            const addListeners = () => {
+              // multiple selects have no focusser
+              if (uiSelect.multiple) {
+                  angular.element(uiSelect.focusInput).on('focus', focusHandler);
+              } else {
+                  angular.element(uiSelect.focusser).on('focus', focusHandler);
+              }
+            }
+            const removeListeners = () => {
+              angular.element(uiSelect.focusInput).off('focus', focusHandler);
+              angular.element(uiSelect.focusser).off('focus', focusHandler);
             }
 
+            addListeners();
```

Comment:
I suggest creating function remove() .. timeout create from `window.addEventListener("focus"` and calling it here instead.